server/models/forum.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const forumSchema = new Schema({
```

Comment:
Just small consistency fix: use pascal case here `ForumSchema`

Reply:
> Quick work! Maybe some other folks can chime in, but we may want to break out Posts and Comments into their own models? Anyone have any points for or against making them models?

Just to clarify, does this mean to have a Post model with comment fields in it or to have separate Posts and Comments model?
I think if we create a Comments model separately it will be a bit easier to handle comment threads.